packages/sanes-chrome-extension/src/routes/signup/components/NewAccountForm.tsx

```diff
@@ -10,10 +10,15 @@ import Form, {
 import TextFieldForm from 'medulas-react-components/lib/components/forms/TextFieldForm';
 import PageLayout from 'medulas-react-components/lib/components/PageLayout';
 import { SIGNUP_ROUTE } from '../../paths';
+import CheckboxField from 'medulas-react-components/lib/components/forms/CheckboxField';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import Link from 'medulas-react-components/lib/components/Link';
 
 export const ACCOUNT_NAME_FIELD = 'accountNameField';
 export const PASSWORD_FIELD = 'passwordInputField';
 export const PASSWORD_CONFIRM_FIELD = 'passwordConfirmInputField';
+export const TERMS_ACCEPT_FIELD = 'termsAcceptCheckboxField';
+export const TERMS_URL = 'https://support.iov.one/hc/en-us';
```

Comment:
I'd say we should put this on `src/routes/paths`, maybe near soon in the future, we create our own view/path.